libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -1079,6 +1164,34 @@ Vector3f AC_PosControl::lean_angles_to_accel(const Vector3f& att_target_euler) c
     };
 }
 
+/// set the horizontal position offset target in cm from EKF origin in NE frame
+void AC_PosControl::set_pos_offset_target_xy_cm(const Vector2p& pos_offset_target_xy_cm)
```

Comment:
Here we would ideally set:
_pos_offset_target
_vel_offset_target
_accel_offset_target

Reply:
So if the user only wanted to set pos they could just leave vel and accel as zero right?